File: dbflow/src/androidTest/java/com/raizlabs/android/dbflow/test/global/GlobalDatabase.java
package com.raizlabs.android.dbflow.test.global;

import com.raizlabs.android.dbflow.annotation.ConflictAction;
import com.raizlabs.android.dbflow.annotation.Database;

/**
 * Description:
 */
@Database(name = GlobalDatabase.NAME, version = GlobalDatabase.VERSION,
        insertConflict = ConflictAction.REPLACE,
        updateConflict = ConflictAction.REPLACE)
public class GlobalDatabase {

    public static final String NAME = "GlobalDatabase";

    public static final int VERSION = 1;
}

File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/test/global/GlobalDatabase.kt
package com.raizlabs.android.dbflow.test.global

import com.raizlabs.android.dbflow.annotation.ConflictAction
import com.raizlabs.android.dbflow.annotation.Database

/**
 * Description:
 */
@Database(name = GlobalDatabase.NAME, version = GlobalDatabase.VERSION, insertConflict = ConflictAction.REPLACE, updateConflict = ConflictAction.REPLACE)
object GlobalDatabase {

    val NAME = "GlobalDatabase"

    val VERSION = 1
}
